src/main/java/frc/robot/command/elevator/TeleopElevatorCommand.java

```diff
@@ -0,0 +1,152 @@
+package frc.robot.command.elevator;
+
+import edu.wpi.first.math.filter.SlewRateLimiter;
+import edu.wpi.first.networktables.GenericPublisher;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.component.ElevatorSetpointManager;
+import frc.robot.component.ElevatorSetpointManager.ElevatorSetpoint;
+import frc.robot.input.MoInput;
+import frc.robot.molib.MoShuffleboard;
+import frc.robot.molib.prefs.MoPrefs;
+import frc.robot.subsystem.ElevatorSubsystem;
+import frc.robot.subsystem.ElevatorSubsystem.ElevatorControlMode;
+import frc.robot.subsystem.ElevatorSubsystem.ElevatorMovementRequest;
+import frc.robot.subsystem.ElevatorSubsystem.ElevatorPosition;
+import java.util.Optional;
+import java.util.function.Supplier;
+
+public class TeleopElevatorCommand extends Command {
+    private static final double ELEVATOR_HARD_STOP_OVERRIDE_TIMEOUT = 1;
+
+    private static final ElevatorSetpoint DEFAULT_SETPOINT = ElevatorSetpoint.STOW;
+
+    private final ElevatorSubsystem elevators;
```

Comment:
nit: there's only one elevator
```suggestion
    private final ElevatorSubsystem elevator;
```